src/plenigo/builders/CheckoutSnippetBuilder.php

```diff
@@ -57,7 +58,7 @@ public function __construct(Product $productToChkOut) {
      * @param string $elementId id of the element the checkout should be inserted into, if this parameter is passed the checkout will be embedded
      * 
      * @return string A Javascript snippet that is compliant with plenigo's Javascript SDK.
-     * @throws CryptographyException When an error occurs during data encoding
+     * @throws \Exception When an error occurs during data encoding
```

Comment:
Its an EncryptionException, the comment in the encryptWithAES method is wrong

Reply:
I added the EncryptionException